libsquoosh/src/emscripten-utils.ts

```diff
@@ -10,10 +10,14 @@ export function pathify(path: string): string {
 export function instantiateEmscriptenWasm<T extends EmscriptenWasm.Module>(
   factory: EmscriptenWasm.ModuleFactory<T>,
   path: string,
+  workerWasm: string = '',
```

Comment:
As far as I can tell, this should be called `workerJS` not `workerWasm`.

Reply:
Good point